internal/v1/api.yaml

```diff
@@ -1619,6 +1619,10 @@ components:
         data:
           type: string
           description: Contents of the file as plain text
+        data_base64:
```

Comment:
What about `data_encoding`? Make it an enum with `plain` and `base64`, `plain` is the default value. This allows us to add more encodings in the future if we ever have such need.

Reply:
Good idea, enum it is.